documentation/book/assembly-cluster-operator.adoc

```diff
@@ -9,18 +9,20 @@
 
 include::common/attributes.adoc[]
 
-[id='deploying-cluster-operator-{context}']
-= Deploying the Cluster Operator
-
-{ProductName} uses the Cluster Operator to deploy and manage Kafka (including Zookeeper) and Kafka Connect
-clusters.
-The Cluster Operator is deployed as a process running inside of the {ProductPlatformName} cluster.
-To deploy a Kafka cluster, a `Kafka` resource with the cluster configuration has to be created within the {ProductPlatformName} cluster.
-Based on the information in that `Kafka` resource,
-the Cluster Operator deploys a corresponding Kafka cluster.
-The format of the `Kafka` resource is described in <<kafka_assembly_details>>.
-
-NOTE: {ProductName} contains example YAML files which make deploying a Cluster Operator easier.
+[id='cluster-operator-{context}']
+= Cluster Operator
+
+{ProductName} uses the Cluster Operator to deploy and manage Kafka (including Zookeeper) and Kafka Connect clusters.
+The Cluster Operator is deployed inside of the
+ifdef::Kubernetes[Kubernetes or]
+OpenShift cluster.
+To deploy a Kafka cluster, a `Kafka` resource with the cluster configuration has to be created within the
+ifdef::Kubernetes[Kubernetes or]
+OpenShift cluster.
+Based on the way the `Kafka` resource was declared, the Cluster Operator deploys a corresponding Kafka cluster.
```

Comment:
_Based on what is declared inside the Kafka resource ..._ would sound better to me.